src/styles/blocks/page.scss

```diff
@@ -0,0 +1,6 @@
+.page {
+  height: 100vh;
```

Comment:
You have set the height of the page to 100vh, which means it will take the full height of the viewport. This is fine if you want your page to be exactly the height of the viewport, but it could cause issues if your content is taller than the viewport as it could cause some of your content to not be visible. If this is the case, you might want to consider using `min-height` instead of `height` to ensure all content is visible.